db.go

```diff
@@ -8,27 +8,38 @@ import (
 	"github.com/go-pg/pg/types"
 )
 
-var tableName = "gopg_migrations"
+var defaultTableName = "gopg_migrations"
 
 func SetTableName(name string) {
-	tableName = name
+	defaultTableName = name
```

Comment:
it might be best to deprecate this method and instead point people to using a custom Group or modifying DefaultGroup.

Reply:
can't this call `DefaultGroup.SetTableName`?

Reply:
This now calls `DefaultGroup.WithTableName`